python/metatensor-torch/metatensor/torch/atomistic/units.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict
+
+import torch
+
+
+@torch.jit.script
+class Quantity:
+    baseline: str
+    """TODO"""
+
+    conversions: Dict[str, float]
+    """TODO"""
+
+    def __init__(self, baseline: str, conversions: Dict[str, float]):
+        self.baseline = baseline
+        self.conversions = conversions
+
+        assert self.baseline in self.conversions
+        assert self.conversions[self.baseline] == 1.0
+
+
+KNOWN_QUANTITIES = {
+    "length": Quantity(
+        baseline="angstrom",
+        conversions={
```

Comment:
This looks a bit over engineering for me and I don't understand why is there a baseline argument. We just define our baseline argument and everything else is converted to this. A but like `scipy.constants`.

Reply:
The baseline is just here to clarify the conversion factors. We could maybe define something nicer based on scipy.constants, but we need this code to be compatible with torchscript.

We don't (and can't) impose units on either the model or the MD engine, so we can not convert everything to internal units. Instead, we need to know about all the possible units at run time to convert between the engine and model units.